src/main/java/com/minecolonies/coremod/blocks/BlockHutField.java

```diff
@@ -132,20 +135,21 @@ public boolean isPassable(final IBlockAccess worldIn, final BlockPos pos)
     }
 
     //todo: remove once we no longer need to support this
-    @SuppressWarnings("deprecation")
+    @NotNull
+    @SuppressWarnings(DEPRECATION)
     @Override
     public AxisAlignedBB getBoundingBox(final IBlockState state, final IBlockAccess source, final BlockPos pos)
     {
         return new AxisAlignedBB((float) START_COLLISION,
-                                  (float) BOTTOM_COLLISION,
-                                  (float) START_COLLISION,
-                                  (float) END_COLLISION,
-                                  (float) HEIGHT_COLLISION,
-                                  (float) END_COLLISION);
+                (float) BOTTOM_COLLISION,
+                (float) START_COLLISION,
+                (float) END_COLLISION,
+                (float) HEIGHT_COLLISION,
+                (float) END_COLLISION);
     }
 
     //todo: remove once we no longer need to support this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1135)